lib/game.rb

```diff
@@ -0,0 +1,34 @@
+class Game
+  attr_reader :player, :comp_choice
+
+  def initialize(player)
+    @player = player
+    @comp_choice = choose
+  end
+
+  def self.create(player)
+    @game = Game.new(player)
+  end
+
+  def self.instance
+    @game
+  end
+
+  def calculate_result
+    actions = {
+      :scissors => {
+        :rock => "Computer wins", :paper => "#{@player.name} wins", :scissors => "It's a draw" },
+      :rock => {
+        :rock => "It's a draw", :paper => "Computer wins", :scissors => "#{@player.name} wins" },
+      :paper => {
+        :rock => "#{@player.name} wins", :paper => "It's a draw", :scissors => "Computer wins" }
+    }
+    actions[@player.choice][@comp_choice]
+  end
```

Comment:
Clean code, good use of string interpolation. If you wish to work on this more in the future, perhaps you could split this method up? `calculate_result` works out who wins, makes the result string, then returns it, which does not comply with SRP.